scpdt/tests/test_pytest_configuration.py

```diff
@@ -1,14 +1,38 @@
 import pytest
-from pathlib import PosixPath
+import os
+from pathlib import PosixPath, Path
 
-
-from . import module_cases, failure_cases, failure_cases_2, stopwords_cases
+from . import module_cases, failure_cases, failure_cases_2, stopwords_cases, local_file_cases
+from scpdt.plugin import copy_local_files
+from scpdt.tests.conftest import config
 
 pytest_plugins = ['pytester']
 
+
+@pytest.fixture(autouse=True)
```

Comment:
For my education: does it mean this fixture is used for all tests in this test file, for all tests in the package, for all tests in the test run? 
We want to limit it I think and only use it where we need it.

Reply:
From my understanding, autouse fixtures are limited to the scope of the module they are defined in unless defined in a `conftest.py` file or a plugin module. So it is safe to say it will not affect other tests outside of the `test_pytest_configuration` module.